pkg/interactor/save_data_test.go

```diff
@@ -0,0 +1,95 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	// ErrSaveDataDB is returned when the database fails to save data
+	ErrSaveDataDB = errors.New("error when trying to save data")
```

Comment:
Awesome formatting! I do suggest, tho, you avoid the word `error`, since we wrap errors and it might end up something like: `Error: error failed to lala: error something bad: error trying to save data`. Also, `DB` seems redundant, given the context -- but that's opinionated. Something along these lines seems more appropriate 👇 

```suggestion
	// ErrSaveData is returned when the database fails to save data
	ErrSaveData = errors.New("failed to save data")
```

Reply:
Done!